dashboards/ec2/ec2_application_load_balancer_detail.sp

```diff
@@ -36,7 +36,7 @@ dashboard "aws_ec2_application_load_balancer_detail" {
     graph {
       type  = "graph"
       title = "Relationships"
-      query = query.aws_alb_graph_relationships
+      query = query.aws_ec2_alb_relationships_graph
```

Comment:
```suggestion
      query = query.aws_ec2_application_load_balancer_relationships_graph
```

I think we should continue using the full resource name/table name in the query for 2 reasons:
1. For consistency, it's easier to always use the full name instead of making up abbreviations
2. On the Hub site, when looking at all of the queries in the mod (https://hub.steampipe.io/mods/turbot/aws_insights/queries), when the query names are sorted, it's better if they show up all together